tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_unsuccessful_file_modification/rule_audit_rules_unsuccessful_file_modification_open_rule_order/ordered_by_arch.pass.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# profiles = xccdf_org.ssgproject.content_profile_ospp
+# remediation = none
+
+cp ./ordered_by_arch_error.rules /etc/audit/rules.d/
```

Comment:
There is bad filename, but I would prefer to construct the audit file with use of grep and those snippets you have one level up.

`grep -h 'arch=b32.*EACCES' ../*.rules`

What do you think?

Reply:
@dahaic Interesting, I have updated test scenario with your idea.